webapp/components/RelativeDateTime/index.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <span>{{ relativeDateTime }}</span>
+</template>
+
+<script>
+import formatRelative from 'date-fns/formatRelative'
+import { enUS, de, nl, fr, pt, es /*, pl*/ } from 'date-fns/locale'
+const locales = {
+  en: enUS,
+  de,
+  nl,
+  fr,
+  es,
+  pt
+  // pl
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-04-12T08:57:12Z" title="Friday, April 12th 2019, 10:57:12 am +02:00">Apr 12, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,33 @@
+<template>
+  <span>{{ relativeDateTime }}</span>
+</template>
+
+<script>
+import formatRelative from 'date-fns/formatRelative'
+import { enUS, de, nl, fr, pt, es /*, pl*/ } from 'date-fns/locale'
+const locales = {
+  en: enUS,
+  de,
+  nl,
+  fr,
+  es,
+  pt
+  //pl
```


What's wrong with Polish? :thinking: 

@ulfgebhardt by the way, I think I missed one test case: What happens, if you have a `i18n.locale()` but this locale is not in the list above? Could you create a test case for it? I would like to have a default in that case.

Reply:
> [<img alt="ulfgebhardt" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-04-12T09:17:14Z" title="Friday, April 12th 2019, 11:17:14 am +02:00">Apr 12, 2019</time>_

---


Polish is currently not implemented on library side.